js/interface.js

```diff
@@ -1571,6 +1580,21 @@ function checkGroupErrors() {
   });
 }
 
+// We set required attribute to 'demo password' only if 'demo user' field is not empty
+function checkDemoUser() {
+  // When google tries to auto-fill 'demo user' field, we checking data from API and delete google auto-fill
+  // if no saved data for this field
+  // To allow a user to use auto-fill from google but disallow google to put the information to the field by itself.
+  // We check how many times use google auto-fill after numerous tries found out that google inserts data to this input
+  // only three times at a row, so, therefore, the fourth time it's a user trying to input information from auto-fill.
+  var $demoUserFiled = $('#fl-store-revDemoUser');
+  if (!userInput && autoFill < 3) {
```

Comment:
Line break before and after the `if` clause

Reply:
@tonytlwu 
Fixed